### ISSUE TITLE
Actionsheet overlay click event in Android ?

### ISSUE THREAD

**Issue by bahadircivelek** [GeekyAnts/NativeBase#1013]
when user click actionsheet overlay, modal closing in android but what is this modal closing event ? i need to know if user click overlay.

**Comment by bahadircivelek** [comment]
upp

**Comment by akhil-geekyants** [comment]
@bahadircivelek you can write your code inside the second parameter of ActionSheet.show()  which is the function called when you choose an option.
```
ActionSheet.show({
   ...
},(buttonIndex) =>{ //code here }
);
```

**Comment by bahadircivelek** [comment]
Overlay and backbutton click not trigger this callback event i crated pull request for it..

**Comment by akhil-geekyants** [comment]
Merged the Pull request #1242 and released.
